apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -1798,8 +1812,7 @@ const selectConversationsEpic: AppEpic = (action$, state$) =>
         ConversationsActions.saveNewConversationSuccess.match(action) ||
         ConversationsActions.importConversationsSuccess.match(action) ||
         ConversationsActions.deleteConversationsComplete.match(action) ||
-        ConversationsActions.addConversations.match(action) ||
-        ConversationsActions.duplicateConversation.match(action),
```

Comment:
Why you removed duplicate conversation selection?

Reply:
because only when `duplicateConversation` will execute `saveNewConversationSuccess` action we change `selectedConversationIds`